lesson_26/api/java/api_app/src/main/java/com/codedifferently/lesson26/web/MediaItemsController.java

```diff
@@ -29,4 +34,31 @@ public GetMediaItemsResponse getItems() {
     var response = GetMediaItemsResponse.builder().items(responseItems).build();
     return response;
   }
+
+  @PostMapping("/items")
+  public MediaItemResponse addItemById(@RequestBody CreateMediaItemRequest createRequest) {
```

Comment:
FYI, there's a `CreateMediaItemResponse` message type that you should probably use.